nbs/tsdataset.ipynb

```diff
@@ -54,7 +54,8 @@
     "#| export\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.                     files_ds: _FilesDataset,
</pre><p>I don't think this needs to be a files dataset anymore, can be just a regular list of strings</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/1049/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>